<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailsOfContactInGroup</class>
 <widget class="QDialog" name="DetailsOfContactInGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QForm
{
	background-color: black;
}</string>
  </property>
  <widget class="QFrame" name="UpperPanel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background-color: lightgray;
    border: 1px solid #31363F;
    border-radius: 20px;
}

QFrame:hover {
    background-color: darkgray;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Heading">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>10</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: darkgray;
	border: none;
	color: #31363F;
}</string>
    </property>
    <property name="text">
     <string>CONTACT BOOK APP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="LeftPanel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>80</y>
     <width>181</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background-color: lightgray;
    border: 1px solid #31363F;
    border-radius: 20px;
}

QFrame:hover {
    background-color: darkgray;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="RemoveContactFromGroupButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #31363F;
    color: white;
    border: none;
    padding: 8px 16px;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: #CED4DA;
    color: #4CAF50;
}</string>
    </property>
    <property name="text">
     <string>REMOVE FROM GROUP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #31363F;
    color: white;
    border: none;
    padding: 8px 16px;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: #CED4DA;
    color: #4CAF50;
}</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="RightPanel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>481</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	background-color: #272829;
	border: 1px solid #31363F;
	border-radius:20px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="OperationHeading">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border: none;
	background-color: lightgray;
	color: #31363F;
}</string>
    </property>
    <property name="text">
     <string>DETAILS OF CONTACT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ContactPicture">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border: none;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="FirstName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border: none;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PhoneNumber">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border: none;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="EmailAddress">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border: none;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="House">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border: none;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Street">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border: none;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="City">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border: none;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Country">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border: none;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LastName">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	border: none;
}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
